tests/moduleTests/pitch.js

```diff
@@ -64,4 +64,32 @@ export default function tests() {
 
     });
 
+    test('music21.pitch.Pitch Equality', assert => {
+        const data = [['a', 'b', 'False'], ['a', 'a', 'True'], ['a#', 'a', 'False'],
+            ['a#', 'b-', 'False'], ['a#', 'a-', 'False'], ['a##', 'a#', 'False'],
+            ['a#4', 'a#4', 'True'], ['a-3', 'a-4', 'False'], ['a#3', 'a#4', 'False']];
+ 
+        for (let i = 0; i < data.length; i++) {
+            const p1 = new music21.pitch.Pitch(data[i][0]);
+            const p2 = new music21.pitch.Pitch(data[i][1]);
+            if (data[i][2] === 'False') {
```

Comment:
Why not use real booleans here, and in your data structure above?

Reply:
I was having issues with the JavaScript linter in webpack, flagging my code as invalid. I've updated this now.